src/charm.py

```diff
@@ -595,31 +180,21 @@ def _action_certificates_are_valid(self, event: ActionEvent) -> bool:
 
         return True
 
-    def get_missing_configuration_options(self) -> List[str]:
-        """Returns the list of missing configuration options.
+    def _relation_created(self, relation_name: str) -> bool:
+        """Returns whether given relation was created.
 
-        Returns:
-            list: List of config option names.
-        """
-        missing_config_options = []
-        required_config_options = [
-            "certificate",
-            "ca-certificate",
-        ]
-        for config in required_config_options:
-            if not self.model.config.get(config, None):
-                missing_config_options.append(config)
-        return missing_config_options
-
-    @staticmethod
-    def _generate_password() -> str:
-        """Generates a random 12 character password.
+        Args:
+            relation_name (str): Relation name
 
         Returns:
-            str: Password
+            bool: True/False
```

Comment:
I would prefer not too have this line at all, it does not say anything more than the type hint and the name and description.

Reply:
✅